libs/contentful-translation-extension/src/components/Button.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react'
+
+import { BaseProps } from './MarkdownEditor'
+
+interface ButtonProps extends BaseProps {
+  active: boolean
+  icon: string
+}
+
+export const ICONS = [
+  { id: 'format_bold', src: require('../assets/bold.png') },
+  { id: 'format_italic', src: require('../assets/italic.png') },
+  { id: 'format_underlined', src: require('../assets/underlined.png') },
+  { id: 'insert_link', src: require('../assets/insert_link.png') },
+  { id: 'looks_one', src: require('../assets/looks_one.png') },
+  { id: 'looks_two', src: require('../assets/looks_two.png') },
+  { id: 'format_list_numbered', src: require('../assets/list_numbered.png') },
+  { id: 'format_list_bulleted', src: require('../assets/list_bulleted.png') },
+]
+
+export const Button: FC<ButtonProps> = ({ active, icon, ...props }) => {
+  const getIcon = ICONS.find((item) => item.id === icon)?.src ?? ICONS[0].src
+
+  return (
+    <button
+      {...props}
+      style={{
```

Comment:
Yes, the extension is using `create-contentful-extension` which is using behind the scene Parcel to manage bundles. I didn't find any easy way to setup Treat using Parcel, instead of webpack with our default setup with NX, so I didn't bother trying to figure it out more and just write plain CSS objects.

This extension is out of our common dev pipeline and is not using any of NX's builders and only rely on contenful's bundle packages